workflows/align_gisaid/align.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -Eeuo pipefail
```

Comment:
This is more just a curiosity question not at all a suggestion, but is there a reason you wrote this workflow in bash instead of python?

Reply:
A monolithic python script would not work if we want to pick up updates to aspen.  We could do a python script that starts up another python script for the save portion.  Not sure that'd be significantly simpler in python, but I guess it could plausibly be more readable.